Keep On Going/kog_class.py

```diff
@@ -229,9 +231,9 @@ def __init__(self, width, height):
         self.range_index = 0
 
         self.diff_lookup = {

```

Comment:
We can designate an int rather than a whole dictionary as it's 1:1 in key to value. In the future if the values are changed, a formula could be used at each location this is used (OptionsPage and Memory) because it increases and decreases proportionally as difficulty value increases.